pages/+page.svelte

```diff
@@ -13,13 +13,17 @@
   import { dev } from '$app/environment';
   import { assets } from '$app/paths';
   import { page } from '$app/stores';
+  import { browser } from '$app/environment';
 
   // Google doc content
   import content from '$locales/en/content.json';
 
   // Styles
   import '@reuters-graphics/graphics-components/scss/main';
   import '$lib/styles/global.scss';
+
+  // Checks if it's on the client side and has access to search params
+  const searchParams = browser && $page.url?.searchParams;
```

Comment:
Could probably just wrap this condition with the one in the `if` svelte tag below to a simple `isReutersMobileApp` variable, I think.